tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -35,15 +34,22 @@
       description="[[tagMetadata.description]]"
     ></tf-card-heading>
 
-    <div id="chart-and-spinner-container">
-      <vz-line-chart
+    <div id="chart-container">
+      <tf-line-chart-data-loader
         x-components-creation-method="[[_xComponentsCreationMethod]]"
         y-value-accessor="[[_yValueAccessor]]"
         tooltip-columns="[[_tooltipColumns]]"
         color-scale="[[_colorScaleFunction]]"
         default-x-range="[[_defaultXRange]]"
         default-y-range="[[_defaultYRange]]"
-      ></vz-line-chart>
+        smoothing-enabled="[[_smoothingEnabled]]"
```

Comment:
Is it possible to just pass `"false"` here?

Reply:
I couldn't get `"false"` to work. I believe this is why: https://stackoverflow.com/a/39855542

Reply:
That is obnoxious, but makes sense.\* Your code is fine with me, then.

\* `<grumble>` as usual, React does the right thing here… `</grumble>`

Reply:
Yeah :/